lib/ansible/modules/storage/netapp/na_ontap_autosupport.py

```diff
@@ -176,20 +221,40 @@ def modify_autosupport_config(self, modify):
                 asup_details.add_new_child('is-support-enabled', 'true')
             elif modify.get('support') is False:
                 asup_details.add_new_child('is-support-enabled', 'false')
+        if modify.get('hostname_subject') is not None:
+            if modify.get('hostname_subject') is True:
+                asup_details.add_new_child('is-node-in-subject', 'true')
+            elif modify.get('hostname_subject') is False:
```

Comment:
it can just be  else:
as the type is bool.

Though I would rather do:
asup_details.add_new_child('is-node-in-subject', 'true' if modify.get('hostname_subject') else 'false' )

Reply:
@lonico, I realize that, however I was just trying to keep the same style as other places in the same module.  I didn't want to change styles in the middle of the module.